[1.0.4 (and older)] File `tests/conftest.py` is arguably missing from source tarballs?

Hi!

I just noticed that running the test from the source tarball `hashin-1.0.4.tar.gz` (but also `hashin-1.0.3.tar.gz`) a la …

```console
cd "$(mktemp -d)"
wget https://files.pythonhosted.org/packages/04/04/9061632da906161ebbc6ebf326267e5be2ce135552666c555272ee07806f/hashin-1.0.4.tar.gz
tar xf hashin-1.0.4.tar.gz 
cd hashin-1.0.4/
python3.13 -m venv venv
source venv/bin/activate
pip3 install -e .
PYTHONPATH=. pytest -v
```

…fails with error …

```
________________________________________________________________________________ ERROR at setup of test_get_latest_version_simple ________________________________________________________________________________
file /tmp/tmp.nul9E8JlHf/hashin-1.0.4/tests/test_cli.py, line 33
  def test_get_latest_version_simple(murlopen):
E       fixture 'murlopen' not found
>       available fixtures: _session_event_loop, anyio_backend, anyio_backend_name, anyio_backend_options, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, event_loop, event_loop_policy, git_repo, make_git_repo, make_repo, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, repo, tests/test_cli.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/tmp/tmp.nul9E8JlHf/hashin-1.0.4/tests/test_cli.py:33
```

…due to missing file `tests/conftest.py`.

Probably the source tarball should either (a) ship all files needed to run the tests or (b) not ship the tests at all (and then distros need to use GitHub-generated Git archives in packaging as a consequence rather than PyPI). @peterbe what do you think?

Best, Sebastian

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,1 @@
+include tests/conftest.py

